airwallex/src/main/java/com/airwallex/android/view/util/AvailablePaymentMethodExtensions.kt

```diff
@@ -1,8 +1,21 @@
 package com.airwallex.android.view.util
 
 import com.airwallex.android.core.model.AvailablePaymentMethodType
+import com.airwallex.android.core.model.PaymentConsent
 import com.airwallex.android.core.model.PaymentMethodType
 
 fun List<AvailablePaymentMethodType>.findWithType(type: PaymentMethodType): AvailablePaymentMethodType? {
     return find { it.name == type.value }
-}
\ No newline at end of file
+}
+
+fun List<AvailablePaymentMethodType>.hasSinglePaymentMethod(
+    desiredPaymentMethodType: AvailablePaymentMethodType?,
```

Comment:
I think this input can just be `PaymentMethodType`, so all the logic to decide whether the method list has a single designated type is wrapped here instead of checking null.

Reply:
ok